docs/how-to/how-to-use-streamk.rst

```diff
@@ -0,0 +1,83 @@
+.. meta::
+   :description: How to use Stream-K with the hipBLASLt library for GEMM operations
+   :keywords: hipBLASLt, ROCm, library, API, tool
+
+.. _streamk:
+
+********************************
+Using Stream-K with hipBLASLt
+********************************
+
+hipBLASLt supports the Stream-K library, which provides improved performance
```

Comment:
Is the performance tested? The library size is smaller for sure, but maybe a perf chart will be better.